app/controllers/stories_controller.rb

```diff
@@ -112,6 +114,7 @@ def handle_user_or_organization_or_podcast_or_page_index
 
   def handle_tag_index
     @tag = params[:tag].downcase
+    @page = (params[:page] || 1).to_i
```

Comment:
This obviously works, but there's also `params.fetch(:page, 1).to_i`, which I think is slightly nicer. No need to change now, but good to know for the future I guess.